code/modules/ftl/machinery/phase_cannon.dm

```diff
@@ -0,0 +1,95 @@
+/obj/machinery/power/shipweapon //PHYSICAL WEAPON
+	name = "phase cannon"
+	desc = "A powerful weapon designed to take down shields.\n<span class='notice'>Alt-click to rotate it clockwise.</span>"
+	icon = 'icons/obj/96x96.dmi'
+	icon_state = "phase_cannon"
+	pixel_x = -32
+	pixel_y = -32
```

Comment:
-23 looks a little cooler but I feel this leads to an argument about consistent 2d style
![dreamseeker_2018-05-14_20-26-01](https://user-images.githubusercontent.com/29457447/40018786-2c8b4606-57b5-11e8-8b97-c75839fe136e.jpg)